src/js/lib/client/src/preload.js

```diff
@@ -30,26 +30,6 @@ window.showAnimation = animate_show(win)
 
 
 
-// ==========================================
-// === [MacOS] Show / Hide Window Buttons ===
-// ==========================================
-
-windowButtonsVisible = false
-win.setWindowButtonVisibility(windowButtonsVisible)
-window.addEventListener('mousemove', e => {
-    if(e.clientY < 36 && e.clientX < 80) {
-        if(!windowButtonsVisible) {
-            windowButtonsVisible = true
-            win.setWindowButtonVisibility(windowButtonsVisible)
-        }
-    } else if (windowButtonsVisible) {
-        windowButtonsVisible = false
-        win.setWindowButtonVisibility(windowButtonsVisible)
-    }
-})
-
-
-
```

Comment:
On non-macOS platforms this should be a no-op.

In case there is no frame, this should still run. Ideally this should be conditional on presence of the frame in the Window.

Reply:
Perhaps `args` from index.js can be accesed or window can be queried for options it was created with / frame presence?